### PR TITLE
feat(components): created loader and pagination components on table

### LINKED ISSUE #67: Apply improvements on tables design
- [x] Table empty state.
- [x] Table loader state.
- [x] Pagination.

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR do?

- Resolve #67

### CODE REVIEW

File: dist/index.css
Comment (leisterfrancisco) [inline dist/index.css:1]: Is dist folder necessary?

File: src/stories/Table.stories.tsx [hunk 1]
```diff
@@ -14,7 +14,21 @@ const Template: Story<ITable> = (args) => {
...
+      nextPage={() => {}}
+      totalRowsNumber={22}
+      rowsPageSelected={5}
+      rowsPage={[5, 10, 15]}
+      handleSelectedRowsPage={() => {}}
+      loaderComponent={
+        <div
+          style={{ width: "100%", display: "flex", justifyContent: "center" }}
+        >
+          <span>cargando...</span>
```
Comment (leisterfrancisco) [inline src/stories/Table.stories.tsx:29]: should it be loaded from i18 solution?

Why spanish?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — COMMENTED** [review]
can we integrate i18n ?

File: src/stories/Table.stories.tsx (hunk 1 above)
Comment (AngeloCG97) [reply src/stories/Table.stories.tsx:29]: These are sample texts for the webapp storybook, not for the library.

File: src/components/Table/index.tsx [hunk 2]
```diff
@@ -76,16 +111,90 @@ export const Table = ({
...
             )}
           </table>
-          {!rows?.length && (
+          <div className="pagination-container">
+            <div className="container-rows-per-page">
+              <span
+                className="variant-body1"
+                style={{ fontFamily: fontFamily }}
+              >
+                Rows per page:
```
[the remark this answers is not in the document]
Comment (AngeloCG97) [reply src/components/Table/index.tsx:130]: I will receive this text as a parameter for the component and have the person who integrates it send the text in the language they want.

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**